src/components/widgets/status/StatusTab.vue

```diff
@@ -18,25 +17,41 @@
           v-if="progressVisible && $vuetify.breakpoint.lgAndUp"
           cols="auto"
           align-self="center"
+          class="position-relative"
         >
-          <v-progress-circular
-            :rotate="-90"
-            :size="90"
-            :width="7"
-            :value="estimates.progress"
-            color="primary"
+          <div
+            class="progress-wrapper"
+            style="cursor: pointer"
+            @click="handleViewThumbnail"
           >
-            <span class="percentComplete focus--text">{{ estimates.progress }}%</span>
-          </v-progress-circular>
+            <v-progress-circular
+              :rotate="-90"
+              :size="90"
+              :width="7"
+              :value="estimates.progress"
+              color="primary"
+              class="progress-circle"
+            />
+            <div class="thumbnail-background" />
+
+            <div class="fill-height print-thumb">
+              <img
+                class="print-thumb"
+                :src="thumbnail"
+              >
+            </div>
+            <div class="percentage-wrapper">
+              <span class="primary--text">{{ estimates.progress }}%</span>
+            </div>
+          </div>
         </v-col>
```

Comment:
Sorry, looks like I forgot a line in my selection
```suggestion
                :src="thumbnail"
              >
            </div>
          </div>
          <div class="percentage-wrapper">
            <span class="primary--text">{{ estimates.progress }}%</span>
          </div>
        </v-col>
```